txpipe/auxiliary_maps.py

```diff
@@ -175,57 +130,129 @@ def finalize_mappers(self, pixel_scheme, mappers):
 
         # Save PSF maps
         for b in psf_mapper.source_bins:
-            maps["aux_maps", f"psf/g1_{b}"] = (pix, g1[b])
-            maps["aux_maps", f"psf/g2_{b}"] = (pix, g1[b])
-            maps["aux_maps", f"psf/var_g2_{b}"] = (pix, var_g1[b])
-            maps["aux_maps", f"psf/var_g2_{b}"] = (pix, var_g1[b])
-            maps["aux_maps", f"psf/lensing_weight_{b}"] = (pix, weight[b])
-
-        # Save depth maps
-        maps["aux_maps", "depth/depth"] = (depth_pix, depth)
-        maps["aux_maps", "depth/depth_count"] = (depth_pix, depth_count)
-        maps["aux_maps", "depth/depth_var"] = (depth_pix, depth_var)
-
-        # Save bright object counts
-        maps["aux_maps", "bright_objects/count"] = (brobj_pix, brobj_count)
+            maps["aux_source_maps", f"psf/g1_{b}"] = (pix, g1[b])
+            maps["aux_source_maps", f"psf/g2_{b}"] = (pix, g1[b])
+            maps["aux_source_maps", f"psf/var_g2_{b}"] = (pix, var_g1[b])
+            maps["aux_source_maps", f"psf/var_g2_{b}"] = (pix, var_g1[b])
+            maps["aux_source_maps", f"psf/lensing_weight_{b}"] = (pix, weight[b])
 
         # Save flag maps
         for i, (p, m) in enumerate(zip(flag_pixs, flag_maps)):
             f = 2 ** i
-            maps["aux_maps", f"flags/flag_{f}"] = (p, m)
+            maps["aux_source_maps", f"flags/flag_{f}"] = (p, m)
             # also print out some stats
             t = m.sum()
             print(f"Map shows total {t} objects with flag {f}")
 
         return maps
 
 
-class TXStandaloneAuxiliaryMaps(TXAuxiliaryMaps):
```

Comment:
There is no `TXStandaloneAuxillaryMaps` any more. Is `TXAuxillaryLensMaps` supposed to substitute this altogether? It doesn't quite look that way to me.

Reply:
`TXStandaloneAuxillaryMaps` was a stub that was never actually filled in, so it never did anything in the end. TXAuxillaryLensMaps is the replacement for it, yes.